docs/core/preview3/tools/dotnet-remove-reference.md

```diff
@@ -0,0 +1,68 @@
+---
+title: dotnet-remove reference command | Microsoft Docs
+description: The dotnet-remove reference command provides a convenient option to remove project to project references.
+keywords: dotnet-remove , CLI, CLI command, .NET Core
+author: spboyer
+ms.author: mairaw
+ms.date: 02/16/2017
+ms.topic: article
+ms.prod: .net-core
+ms.technology: dotnet-cli
+ms.devlang: dotnet
+ms.assetid: 889c6b7e-a313-40b1-9fd3-6a6f4c52f1d0
+---
+# dotnet-remove reference
+
+[!INCLUDE[preview-warning](../../../includes/warning.md)]
+
+## Name
+
+`dotnet-remove reference` - Remove project to project references.
+
+## Synopsis
+
+```
+dotnet remove [<PROJECT>] reference [-f|--framework] [args] [-h|--help]
+dotnet remove reference [-h|--help]
+```
+
+## Description
+
+The `dotnet remove reference` command provides a convenient option to remove a project references from a project.
```

Comment:
I added this to the dotnet-add-reference.md as it seems to make more sense there.